src/compiler/transformers/es2015.ts

```diff
@@ -2867,7 +2904,6 @@ export function transformES2015(context: TransformationContext): (x: SourceFile
 
         // Disable trailing source maps for the OpenParenToken to align source map emit with the old emitter.
         setEmitFlags(forStatement, EmitFlags.NoTokenTrailingSourceMaps);
-        setTextRange(forStatement, node);

```

Comment:
this was redundant - this call is a mutation and the same call is already done when assigning to `forStatement`